### PR TITLE
fix: Miscellaneous changes/fixes to content copy/images

### LINKED ISSUE #6: Miscellaneous changes/fixes to content copy/images
## Tasks

**Change every instance of the term ZEBEDEE for ZBD:**
- `ZEBEDEE` becomes `ZBD`
- `ZEBEDEE API` becomes `ZBD API`
- Anything that is `ZEBEDEE ______` should become `ZBD ______`
- If something is `ZBD _____`, leave it as is
- If it's a name of a package or library (e.g. `zebedee-rust` crate, etc), do not change it
- `ZEBEDEE Developer Dashboard` should becomes `ZBD Dev Dashboard`

**Change logo to ZBD:**
- Dark Mode
ZBD_LogotypeLight

- Light Mode
ZBD_LogotypeDark

**Understand and fix this crashing issue when clicking on the API References:**
- See video: https://www.dropbox.com/scl/fi/wo7hjmm86tnytfvdbjh0f/Screen-Recording-2023-08-13-at-8.25.13-PM.mov?rlkey=r2durehp2x0qtfy3ql8aur8ik&dl=0

**Change favicon to this new one:**
favicon

**Change Menu Item from `Status` to `API Status`**

**Change Menu item from pointing to `status.zebedee.io` to point to `status.zbd.dev`** (domain is still recycling, don't worry if this URL doesn't render yet)

### PR DESCRIPTION
**Pull request by rishi-raj-jain**
fix #6
/claim #6

### CODE REVIEW

**Comment by rishi-raj-jain** [comment]
@andrerfneves this is ready for review.

File: docs/dashboard/projects/sandbox.mdx [hunk 1]
```diff
@@ -56,39 +56,39 @@ Now that we have a clear understanding of Sandbox Projects and Sandbox APIs, let
...
-  <img src="/img/v2/devdash/sandbox-api-key.png" alt="ZEBEDEE Developer Dashboard | Sandbox API Key" />
+  <img src="/img/v2/devdash/sandbox-api-key.png" alt="ZBD Dev Dashboard | Sandbox API Key" />
 </Frame>
 <br />
 
 Now that we have our Sandbox API Key, let's use the [**Send Bitcoin to ZBD Gamertag Sandbox API**](/api/sandbox/gamertag/send-payment) to send some _fake satoshis_ to our Sandbox Test User.
 
-You may write your own HTTP client to do this test, but the simplest way is to use the _API Playground Module_ available in the [ZEBEDEE API Reference](https://docs.zebedee.io/api). On every API Reference page you will find a _ZEBEDEE API Playground Module_ on the right-hand side that you can use to directly test an API endpoint by providing it an API Key and a payload (if necessary).
+You may write your own HTTP client to do this test, but the simplest way is to use the _API Playground Module_ available in the [ZBD API Reference](https://docs.zbd.dev/api). On every API Reference page you will find a ZBD API Playground Module_ on the right-hand side that you can use to directly test an API endpoint by providing it an API Key and a payload (if necessary).
```
Comment (andrerfneves) [inline docs/dashboard/projects/sandbox.mdx:65]: missing `_` before ZBD in this case

Comment (rishi-raj-jain) [reply docs/dashboard/projects/sandbox.mdx]: Done

**Comment by andrerfneves** [comment]
Thanks @rishi-raj-jain. Love the velocity here! I'll have more tomorrow ;)

**Comment by rishi-raj-jain** [comment]
@andrerfneves 

Appreciate your kind words!

Sure thing, feel free to tag me.

**Pull request merged**